client/src/components/settings/toggle-setting.css

```diff
@@ -80,50 +75,46 @@
 
 .toggle-radio-group {
   display: flex;
-  align-items: start;
 }
 
 .toggle-radio-group label {
-  font-weight: normal;
-  display: flex;
-  align-items: center;
+  margin: 8px;
 }
 
 .toggle-radio-group input {
-  -webkit-appearance: none;
-  appearance: none;
-  height: 0.8rem;
-  width: 0.8rem;
-  border-radius: 50%;
-  background: transparent;
-  border: 2px solid var(--secondary-color);
-  margin: 0;
-}
-
-.toggle-radio-group [data-checked='true'] input {
-  background: var(--secondary-color);
-}
-
-.toggle-radio-group [data-checked='true'] span {
-  font-weight: 700;
-}
-
-.toggle-radio-group input:focus-visible {
-  outline-offset: 1px;
+  position: absolute;
+  left: -9999px;
```

Comment:
This code, is deleting the focus for keyboard user, turn `#radioAisLocked` to a grid, and stack the `input` and the `span` in the same are, using `grid-columns` or `grid-area` should do the trick and keep the visual keyboard focus

![image](https://user-images.githubusercontent.com/88248797/233677560-a484bdbd-63a0-42ce-8684-2f711d42607b.png)

Reply:
I'm not sure this is the way to go. We are using custom elements for the radio buttons so the focus indicator should go around those elements. I don't think we want to try to use the indicator around the `input` since it may not match up perfectly and it is a more complicated solution. The solution here is to just add a focus indicator around `.custom-circle` when the `input` it is associated with has focus. This can easily be done with `.toggle-radio-group input:focus + .custom-circle`.

Reply:
interesting, I didn't know that was possible, because whenever I press tab, I expected it to navigate to the interactive element which is the `input`, if it is possible, of course go for it

Reply:
You are still tabbing to the actual `input`, but visually the span that we are using to represent the radio button will get a focus ring around it. So we are just hiding the `input` from view but it is still working as normal.

Reply:
Using `.toggle-radio-group input:focus-visible + .custom-circle` works!! I tried `.toggle-radio-group input:focus-visible .custom-circle`, but why do we need `+` for this? Trying to understand it.